src/pybind/mgr/ansible/module.py

```diff
@@ -338,14 +432,86 @@ def remove_osds(self, osd_ids):
                                                  params = extravars)
 
         # Filter to get the result
-        ansible_operation.process_output = process_playbook_result
+        ansible_operation.process_output =  process_playbook_result
         ansible_operation.event_filter = "playbook_on_stats"
 
         # Execute the playbook
         self._launch_operation(ansible_operation)
 
         return ansible_operation
 
+    def add_host(self, host):
+        # type: (str, str) -> WriteCompletion
+        """
+        Add a host to the Ansible Runner Service inventory in the "orchestrator"
+        group
+
+        :param host: hostname
+        """
+
+        url_group = URL_MANAGE_GROUP.format(group_name = ORCHESTRATOR_GROUP)
+
+        try:
+            # Create the orchestrator default group if not exist.
+            # If exists we ignore the error response
+            dummy_response = self.ar_client.http_post(url_group, "", {})
+
+            # Here, the default group exists so...
+            # Prepare the operation for adding the new host
+            add_url = URL_ADD_RM_HOSTS.format(host_name = host,
+                                              inventory_group = ORCHESTRATOR_GROUP)
+
+            operations =  [HttpOperation(add_url, "post")]
+
+        except Exception as ex:
```

Comment:
see above.

Reply:
- Included return information in docstring.
- Error management changed